tests/installation/encrypted_volume_activation.pm

```diff
@@ -0,0 +1,27 @@
+use base "basetest";
+use strict;
+use testapi;
+
+sub run {
+    assert_screen 'encrypted_volume_activation_prompt';
+    if (get_var('ENCRYPT_CANCEL_EXISTING')) {
+	send_key 'alt-c', 5;
```

Comment:
well, after the previous assert_screen you can be sure the screen is shown. Then alt-c is pressed and the following check_screen will continue to look at the screen for 30 seconds and check if the prompt reappears. I don't see this send_key critical but the choice of timeout on the next screen can be. That is also why I suggested in the next comment to better do an assert_screen with multiple tags and not a single check screen.